src/splitInteger.test.js

```diff
@@ -2,20 +2,20 @@
 
 const splitInteger = require('./splitInteger');
 
-test(`should split a number into equal parts
+it(`should split a number into equal parts
   if a value is divisible by a numberOfParts`, () => {
-
+  expect(splitInteger(2, 2)).toEqual([1, 1]);
 });
 
-test(`should return a part equals to a value
+it(`should return a part equals to a value
   when splitting into 1 part`, () => {
-
+  expect(splitInteger(17, 4)).toHaveLength(4);
```

Comment:
It says "when splitting into 1 part" and you have numberOfParts (the second argument) 4. You should be checking how 17 is divided by 1.